### DISCUSSION TITLE
Current DBeaver appears to require Java 17 0penjdk

### DISCUSSION THREAD

**Discussion by ghmorrison** [dat-linux/community#68] [Issues]
After trying to open DBeaver I get the following error:

A Java Runtime Environment (JRE) or Java Development Kit (JDK)
must be available in order to run DBeaver. No Java virtual machine was found after searching the following locations:
/usr/lib/jvm/java-17-openjdk-amd64/bin/java

It would have been nice to use it but it is not crucial. I am just sending it in to let you know.

**Comment by dat-linux** [comment]
It seems DBeaver latest versions require Java 17 (whereas before it was Java 11 - which is installed by default). Please just install Java 17 and you should be good to go.
```

sudo apt-get update
sudo apt install openjdk-17-jdk openjdk-17-jre

```
